src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/prisonersearch/PrisonerSearchResultsLogger.kt

```diff
@@ -0,0 +1,84 @@
+package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.prisonersearch
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.prisonersearch.model.PagePrisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.model.PrisonerMatches
+
+private val log = KotlinLogging.logger {}
+
+@Component
```

Comment:
All of the messy (untested!) logging that was previously in `PrisonerSearchService` has been moved here
Its still untested, but was moved verbatim (except 1 addition highlighted below); and is very much throw away code - once we have done the analysis it will be easy to unpick the logging simply by deleting this class and the calls to it 👍